### PR TITLE
Nuxt support

### PR DESCRIPTION
**Pull request by daniarsuprianto**
This PR will merge ui components and nuxt components (in separate nuxt app). This way it becomes tree-shakable and can be imported automatically
https://nuxtjs.org/blog/improve-your-developer-experience-with-nuxt-components/#third-party-component-library

### CODE REVIEW

**Comment by jpsc** [comment]
Couple questions:

Does this mean that we no longer need to create this file?

```javascript
// ~/plugins/funda-ui.js
import Vue from 'vue'
import { UiInput, UiButton, UiIcon } from '@funda/ui'

Vue.component('UiInput', UiInput)
Vue.component('UiButton', UiButton)
Vue.component('UiIcon', UiIcon)

```

Will this change work if you don't use `components: true` on `nuxt.config.js`? Because we have been disabling this for now.

**Comment by daniarsuprianto** [comment]
@jpsc 
> Does this mean that we no longer need to create this file?
> 
> ```js
> // ~/plugins/funda-ui.js
> import Vue from 'vue'
> import { UiInput, UiButton, UiIcon } from '@funda/ui'
> 
> Vue.component('UiInput', UiInput)
> Vue.component('UiButton', UiButton)
> Vue.component('UiIcon', UiIcon)
> ```
Yes, this file is no longer needed, but need to set `components: true`.

> Will this change work if you don't use `components: true` on `nuxt.config.js`? Because we have been disabling this for now.

I think it will work, as long as we don't import this file into nuxt.config.js in the buildModules.
```js
buildModules: ["@funda/ui/src/nuxt.js"]
```
im curious why do we disable the `components: true`?

File: README.md [hunk 1]
```diff
@@ -4,5 +4,37 @@
...
+```
+
+If you are using [Nuxt auto import component](https://nuxtjs.org/docs/2.x/directory-structure/components):
+
+```js
+// nuxt.config.js
+
+export default {
+  ...,
+  components: true,
```
Comment (dine) [inline README.md:32]: I think we don't need this extra file:
https://git.funda.nl/projects/WEBAPI/repos/makelaarsearch.funda.io/browse/nuxt.config.js#19

Comment (daniarsuprianto) [reply README.md:32]: you can just set it to true, yes. but this PR is up for discussion, if we want to do it like yours, or use this config

**Comment by jpsc** [comment]
> @jpsc 
> im curious why do we disable the `components: true`?

Just because it's one of those things you need to know it exists. And since it was a new feature of Nuxt it would make easier to be explicit when using a component.

**Review by jpsc — APPROVED** [review]

**Pull request merged**